Log/lambda_function.py

```diff
@@ -36,9 +36,16 @@
     "ddsourcecategory": "aws",
 }
 
+try:
+    host = os.environ['DD_URL']
```

Comment:
Just an observation, you can simplify the code by using os.getenv and setting the default if it's missing
`host = os.getenv("DD_URL", default="lambda-intake.logs.datadoghq.com")`